sensors:

  logical_camera_1:
    type: logical_camera
    pose:
      xyz: [3.1, 1.85, 1.7]
      rpy: [0, 1.5707, 0]
      
  rgbd_camera_1:
    type: rgbd_camera
    pose:
      xyz: [3.1, 1.8, 1.75]
      rpy: [0, 'pi', 0]

  logical_camera_2:
    type: logical_camera
    pose:
      xyz: [3.1, 3.8, 2.4]
      rpy: [0, 'pi/2', 0]

